docs/src/pages/components/grid.mdx

```diff
@@ -24,11 +35,11 @@ with default settings.
 
 ```tsx live
 <Grid debug>
-    <Cell width={4}> I'm cell 1.</Cell>
-    <Cell width={4}> I'm cell 2.</Cell>
-    <Cell width={4}> I'm cell 3.</Cell>
-    <Cell width={6}> I'm cell 4.</Cell>
-    <Cell width={6}> I'm cell 5.</Cell>
+    <Grid.Cell width={4}> I'm cell 1.</Grid.Cell>
```

Comment:
This could also be destructured at the top of the file to prevent this verbosity.

Reply:
Variables can't be declared in the active code editor to my knowledge.